src/js/components/covid19/AmountsVisualization.jsx

```diff
@@ -0,0 +1,858 @@
+                            <text
+                                ref={_obligationQuestion}
+                                className={obligationQuestionData.className}
+                                x={obligationQuestionData.x}
+                                y={obligationQuestionData.y}>
+                                {obligationQuestionData.text}
+                            </text>
+                        </g>
+                    }
+                    {
+                        obligationValueData &&
+                        <g tabIndex="0" aria-label={obligationValueData.text}>
+                            <desc>{obligationValueData.text}</desc>
+                            <text
+                                ref={_obligationValue}
+                                className={obligationValueData.className}
+                                x={obligationValueData.x}
+                                y={obligationValueData.y}>
+                                {obligationValueData.text}
+                            </text>
+                        </g>
+                    }
+                    {
+                        obligationLabelData &&
+                        <g tabIndex="0" aria-label={obligationLabelData.text}>
+                            <desc>{obligationLabelData.text}</desc>
+                            <text
+                                ref={_obligationLabel}
+                                className={obligationLabelData.className}
+                                x={obligationLabelData.x}
+                                y={obligationLabelData.y}>
+                                {obligationLabelData.text}
+                            </text>
+                        </g>
+                    }
+                    {
+                        zeroPercentData &&
+                        <g tabIndex="0" aria-label="Text representing 0%">
+                            <desc>Text representing 0%</desc>
+                            <text
+                                ref={zerPercentRef}
+                                x={zeroPercentData.x}
+                                y={zeroPercentData.y}>
+                                0%
+                            </text>
+                        </g>
+                    }
+                    {
+                        oneHundredPercentData &&
+                        <g tabIndex="0" aria-label="Text representing 100%">
+                            <desc>Text representing 100%</desc>
+                            <text
+                                ref={oneHundredPercentRef}
+                                x={oneHundredPercentData.x}
+                                y={oneHundredPercentData.y}>
+                                100%
+                            </text>
+                        </g>
+                    }
+                </svg>
+            }
+        </div>
+    );
+};
+
+AmountsVisualization.propTypes = propTypes;
```

Comment:
As mentioned in our walkthrough, I would like to break this up into smaller files / make some of the elements and hooks more reusable in a future iteration. This is good for MVP!